app/views/templates/_news_and_documents.html.erb

```diff
@@ -26,11 +26,15 @@
         if articles.any? %>
           <articles>
 
-            <% articles.each do |a| %>
+            <% articles.each do |a| %>c
 
               <article-listing category="<%= cms_block_content(:article_type, a) %>" ref="article">
                 <h3><%= link_to a.label, a.url(:relative) %></h3>
-                <%= image_tag cms_block_content(:image, a).file.url(:small), alt: a.label %>
+
+                <% if(!cms_block_content(:image, a).to_s.strip.empty?)
```

Comment:
I'm not sure this will work, but it's a good idea to make sure the if and end are seperate like 

```
<% if true %>
    <%= do_a_thing %>
<% end %>
```

But we can use `unless` here instead of if...

```
<% unless cms_block_content(:image, article).try(:file).try(:url, :small).blank?
    <%= image_tag cms_block_content(:image, article).file.url(:small), alt: article.label %>`
<% end %>
```